src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -389,10 +423,31 @@ def validate_max_surge(namespace):
         raise CLIError("--max-surge should be an int or percentage")
 
 
+def validate_addons(namespace):
+    if not hasattr(namespace, 'addons'):
+        return
+    addons = namespace.addons
+    addon_args = addons.split(',')
+
+    for addon_arg in addon_args:
+        if addon_arg not in ADDONS:
+            matches = _fuzzy_match(addon_arg, list(ADDONS))
+            matches = str(matches)[1:-1]
```

Comment:
Question: what does this do? Are you removing the "[" and the "]"?

Reply:
Yea, removing the brackets.